src/include/cell.h

```diff
@@ -77,11 +78,13 @@
 #define WT_CELL_64V 0x04         /* Associated value */
 #define WT_CELL_SECOND_DESC 0x08 /* Second descriptor byte */
 
-#define WT_CELL_TS_DURABLE 0x01 /* Newest-durable timestamp */
-#define WT_CELL_TS_START 0x02   /* Oldest-start timestamp */
-#define WT_CELL_TS_STOP 0x04    /* Newest-stop timestamp */
-#define WT_CELL_TXN_START 0x08  /* Oldest-start txn ID */
-#define WT_CELL_TXN_STOP 0x10   /* Newest-stop txn ID */
+#define WT_CELL_PREPARE 0x01          /* Part of prepared transaction */
+#define WT_CELL_TS_DURABLE_START 0x02 /* Start durable timestamp */
+#define WT_CELL_TS_DURABLE_STOP 0x04  /* Stop durable timestamp */
+#define WT_CELL_TS_START 0x08         /* Oldest-start timestamp */
+#define WT_CELL_TS_STOP 0x10          /* Newest-stop timestamp */
+#define WT_CELL_TXN_START 0x20        /* Oldest-start txn ID */
+#define WT_CELL_TXN_STOP 0x40         /* Newest-stop txn ID */
```

Comment:
Is there any obvious grouping we could use to avoid eating all of the bits?

Or is it perfectly reasonably that any random group of these values can be set?

Reply:
The only thing I can think of is that a lack of start or stop timestamp should imply a lack of durable start or stop. I can't see how that can be used to compress this any further though.

Reply:
kk, thanks @tetsuo-cpp.